modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -0,0 +1,41 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
+= Deleting the {sandboxed-containers-operator} deployment custom resource
+
+To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. From the {product-title} web console, select `openshift-sandboxed-containers` from the *Projects* list.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Click *{sandboxed-containers-first}*.
+. Click the *{sandboxed-containers-operator}* tab.
+. Click the scroll-down list in the *Operator Details*, and click *Delete `KataConfig`*.
+. Click *Delete* in the confirmation window.
+. Delete all `pod`, `deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, and `StatefulSet` objects that use `kata` as `runtimeClass`.
++
+[NOTE]
+====
+You can use the OpenShift CLI (`oc`) to retrieve all pods that use `kata` as `runtimeClassName` by running the following command:
```

Comment:
I think for the console section we should limit the use of `oc`. We should show the pods from the console instead.

Reply:
example As an administrator in the console do: 

- Click workloads
- Click Pods
- ...

Reply:
@zanetworker - Can I get clear instructions on how to do this in the console? If someone could post that then I can add it to the doc.